src/workflow/utils.ts

```diff
@@ -55,13 +56,26 @@ export class WorkflowUtils {
     }
   }
 
+  private static async getModuleExportsFromProvider(
+    modulePath: string,
+    provider: WorkflowBindingProvider,
+  ): Promise<any> {
+    try {
+      return await provider.provide(modulePath);
+    } catch (error: any) {
+      // Ignoring error
```

Comment:
If provider import fails anything we can do?

Reply:
Maybe add a test scenario

Reply:
this is intentional as it will fallback to root path and then fail